### DISCUSSION TITLE
Visual Studio Code Tasks.json.

### DISCUSSION THREAD

**Discussion by h4rldev** [HaxeFlixel/flixel#2573] [Q&A]
I am trying to setup the `tasks.json` in Visual Studio Code to run lime test and lime build of the current project.

Visual Studio Code keeps ignoring the task and proceeds to reply with this error:
`Error: The lime task detection didn't contribute a task for the following configuration`

##### Configuration:
```
{
	"version": "2.0.0",
	"tasks": [
		{
			"type": "lime",
			"command": "lime test",
			"group": {
				"kind": "build",
				"isDefault": true
			}
		},
		{
			"type": "lime",
			"command": "lime build",
			"group": {
				"kind": "build",
				"isDefault": false
			}
		}
	]
}
```

Edit: The formatting is really weird in this post

**Comment by Geokureli** [comment]
Your commands are "lime test" and "lime build" when it should just be "test" and "build". Check out the flixel-template tasks.json:

https://github.com/HaxeFlixel/flixel-templates/blob/dev/ide-data/visual-studio-code/.vscode/tasks.json

**Reply by h4rldev** [reply]
Yeah It doesn't work with just test and build either, same message.

**Reply by h4rldev** [reply]
![image](https://user-images.githubusercontent.com/98224660/171376424-0ed614b6-b834-4afc-a4be-fb6ef414ecfd.png)

**Reply by Geokureli** [reply]
do you have the lime extension installed on VSC?

**Reply by h4rldev** [reply]
I do

**Reply by h4rldev** [reply]
![image](https://user-images.githubusercontent.com/98224660/171485780-5d5e6c68-64e1-4378-b5a9-7896cbfee950.png)

**Comment by h4rldev** [comment]
![image](https://user-images.githubusercontent.com/98224660/171400304-b3a452fa-c6e9-4cc0-9e82-432741d53320.png)
